src/backend/src/prisma-query-args/work-package-template.query-args.ts

```diff
@@ -0,0 +1,15 @@
+import { Prisma } from '@prisma/client';
+
+export const workPackageTemplateQueryArgs = Prisma.validator<Prisma.Work_Package_TemplateArgs>()({
+  include: {
+    userCreated: true,
+    userDeleted: true,
+    blockedBy: true
+  }
+});
+
+export const blockedByInfoQueryArgs = Prisma.validator<Prisma.Blocked_By_InfoArgs>()({
+  include: {
```

Comment:
you can remove this include because we don't need to return the work package template on a blocked by